Classes/Controller/BackendServiceController.php

```diff
@@ -208,9 +199,8 @@ public function publishAllAction(): void
     /**
      * Publish nodes
      *
-     * @param array $nodeContextPaths
+     * @param list<string> $nodeContextPaths
```

Comment:
Why `list` here instead of array? AFAIS it is not necessary and would simply set stronger constraints than needed.

Nothing blocking, I'm just currious,

Reply:
phpstan wants us to specify the subtype of the array. And yes indeed we could also use `array<string>` or `string[]` - i think an integer list conveys best the intention.